src/Microsoft.AspNet.PageExecutionInstrumentation.Interfaces/IBufferedTextWriter.cs

```diff
@@ -19,14 +20,16 @@ public interface IBufferedTextWriter
         /// <summary>
         /// Copies the buffered content to the <paramref name="writer"/>.
         /// </summary>
-        /// <param name="writer">The writer to copy the contents to.</param>
-        void CopyTo(TextWriter writer);
+        /// <param name="writer">The <see cref="TextWriter"/> to copy the contents to.</param>
+        /// <param name="encoder">The <see cref="IHtmlEncoder"/> which encodes the content to be written.</param>
+        void CopyTo(TextWriter writer, IHtmlEncoder encoder);
 
         /// <summary>
         /// Asynchronously copies the buffered content to the <paramref name="writer"/>.
         /// </summary>
-        /// <param name="writer">The writer to copy the contents to.</param>
+        /// <param name="writer">The <see cref="TextWriter"/> to copy the contents to.</param>
+        /// <param name="encoder">The <see cref="IHtmlEncoder"/> which encodes the content to be written.</param>
         /// <returns>A <see cref="Task"/> representing the copy operation.</returns>
-        Task CopyToAsync(TextWriter writer);
+        Task CopyToAsync(TextWriter writer, IHtmlEncoder encoder);
```

Comment:
We'll need to let the BrowserLink folks know if any API changes are made in this project.